lib/locater.js

```diff
@@ -7,52 +7,56 @@
 function findIndices(target, line) {
   var indices = [];
 
-  if (typeof target === 'string') {
-    var index = 0;
-    var offset = 0;
```

Comment:
Converting all calls here to regex to reduce code and remove if else statements.